[WIP] Add PX4 advanced tuning info.

hamishwillee opened the pull request:
Fixes #136 
(ie adds support tor PX4 advanced tuning https://github.com/mavlink/qgroundcontrol/pull/6295)

Note, I may split into separate ArduPilot and PX4 docs, which is why this is still WIP.

DonLakeFlyer commented on en/SetupView/Tuning.md:
```diff
@@ -2,22 +2,25 @@
 
-* Select which axes you would like to tune. Tuning all axes at once can take a significant amount of time which may cause you to run out of battery. In order to prevent this you can select to tune only one axis at a time. 
-* Assign AutoTune to one of your transmitter switches. Ensure that switch is in low position before taking off.
+* Select which axes you would like to tune. 
```
It hasn't changed. Why remove those two bullets?

hamishwillee replied:
I _moved_ the bullets. The first one is immediately below and is now a **Tip** (visually more obvious). The second is still there, below the tip.

hamishwillee replied:
This section now moved, but you can see my point above in https://github.com/mavlink/qgc-user-guide/pull/175/commits/573e2d3dc1c3b35168154633e0c9a744cdb46bdf

DonLakeFlyer replied:
Ah, got it

DonLakeFlyer commented on en/SetupView/Tuning.md:
```diff
@@ -29,13 +32,37 @@ Performing an AutoTune:
+
+### Advanced Tuning
+
+> **Note** Advanced tuning is currently only available on PX4 Multicopter frames. <!-- Still only multirotors? -->
```
Multi-Rotor only. It's still a large WIP. I'm liklely to pull if out this next Stable and then turn it back on for new dailies.

hamishwillee commented on en/SetupView/Tuning.md:
```diff
@@ -2,22 +2,25 @@
+## ArduCopter Tuning
 
-A set of Basic Tuning sliders are suppored for both firmwares. They allow you to adjust the specified flight characteristic by moving the slider to the left or right.
+![ArduCopter Tuning Page](../../assets/setup/tuning/arducopter.png)
```
Yes, but is it the same screen for all cases?

hamishwillee commented:
@DonLakeFlyer This is now split into ardupilot and px4 specific docs. I've added the relevant frames for each bit of functionality in the headings.

Ready to merge IMO - OK?

DonLakeFlyer commented:
LGTM
